sm/sm.go

```diff
@@ -22,15 +22,25 @@ import (
 	"fmt"
 
 	"github.com/Peripli/service-manager/api"
+	"github.com/Peripli/service-manager/rest"
 	"github.com/Peripli/service-manager/server"
 	"github.com/Peripli/service-manager/storage"
 	"github.com/Peripli/service-manager/storage/postgres"
 	"github.com/sirupsen/logrus"
 )
 
+// Parameters contains context, environment for configuring a Service Manager server and optional extensions API
+type Parameters struct {
```

Comment:
Settings or Options

Reply:
Package name "app" would be better

Reply:
There are already too much Settings and Options. Parameters makes it clear that it contains optional function parameters.